processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -612,4 +615,19 @@ public boolean isAbstract() {
     public boolean isUpdateMethod() {
         return getMappingTargetParameter() != null;
     }
+
+    public Map<String, Accessor> getGetSourceReadAccessors() {
```

Comment:
It looks like your last version was reverted to before you moved the `getSourceReadAccessors` to the `Method` interface.